### PR TITLE
:zap: Add husky for pre-commit lint

### PR DESCRIPTION
**Pull request by nilooy**
Add husky for pre-commit lint check and prettier fix.

Source: https://blog.logrocket.com/build-robust-react-app-husky-pre-commit-hooks-github-actions/

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,5 +1,34 @@
 const { merge } = require('lodash')
 
 module.exports = merge(require('@tstt/eslint-config/index.js'), {
-  globals: { Meteor: 'readonly', i18n: 'readonly' },
+ env: {
```
Comment (leonardoventurini) [inline .eslintrc.js]: I think some are already present here

https://github.com/techstertech/eslint-config/blob/main/.eslintrc.js

File: src/Analytics.ts [hunk 2]
```diff
@@ -144,6 +148,7 @@ export class Analytics {
     if (label) params.el = label
     if (value) params.ev = value
 
+    // eslint-disable-next-line no-console
```
Comment (leonardoventurini) [inline src/Analytics.ts:147]: I think you can turn this into a warning instead, no need to disallow it I think

**Review by leonardoventurini — APPROVED** [review]
Later I can update `@tstt/eslint-config` with the new settings too 🚀

File: .eslintrc.js (hunk 1 above)
Comment (nilooy) [reply .eslintrc.js]: I removed the duplicate ones

**Pull request merged**